python/src/scipp/plotting/tools.py

```diff
@@ -131,8 +132,8 @@ def find_log_limits(x):
         [vmin, vmax] = find_linear_limits(x)
         if vmin.value <= 0.0:
             if vmax.value <= 0.0:
-                vmin = scalar(0.1)
-                vmax = scalar(1.0)
+                vmin = full_like(vmin, 0.1)
```

Comment:
This looks like a more sensible way to acquire the units and dtype. I assume that is the root of the issue here.